code/__DEFINES/construction.dm

```diff
@@ -102,7 +104,7 @@
 #define CAT_SANDWICH	"Sandwiches"
 #define CAT_SOUP	"Soups"
 #define CAT_SPAGHETTI	"Spaghettis"
-#define CAT_FISH	"Fish"
+#define CAT_SWEETS	"Sweets"
```

Comment:
Being a super-category of most cakes, pies and pastries, it is likely to cause some confusion (specially with the latter).
Just rename the "Pastries" category to "Confectioneries".